src/pages/management/ItemPage.tsx

```
@@ -177,21 +201,24 @@ const ItemPage = () => {
-    setIsModalOpen(true);
-  };
+  const onClickTableRow = useCallback(
+    (row: UsageData) => {
+      setModalTitle(MODAL_TITLES.EDIT);
+      const addInitialInputs = inputOption[MODAL_TITLES.EDIT]?.map((item) => {
+        return {
+          ...item,
+          initial: item.key === "itemId" ? row.id : row[item.key],
+        };
+      });
+      addInitialInputs && setModalInputs(addInitialInputs);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)